src/api/v1/identities/handlers/unlink-device-from-identity.ts

```diff
@@ -63,6 +66,35 @@ export const unlinkDeviceFromIdentity = async (
       return;
     }
 
+    const identitiesOnDeviceToUnlink = await prisma.identitiesOnDevice.findMany(
+      {
+        where: {
+          deviceId,
+          identityId,
+        },
+      },
+    );
+
+    // Unregister from notifications. We make it allSettled because if this fails, it's not a big deal.
+    // Because when sending a noticicatin, we chekc for identity on device anyway and if we can't find it, we also remove it there.
```

Comment:
```suggestion
    // Because when sending a notification, we check for identity on device anyway and if we can't find it, we also remove it there.
```